cmake/CMakeLists.txt

```diff
@@ -171,8 +171,19 @@ if(ENABLE_KSPACE)
 endif()
 
 if(ENABLE_MISC)
-  option(LAMMPS_XDR "include XDR compatibility files for doing particle dumps in XTC format" OFF)
-  if(LAMMPS_XDR)
+  find_path(XDR_INCLUDE_DIR rpc/xdr.h)
+  include(FindPackageHandleStandardArgs)
+  find_package_handle_standard_args(XDR_HEADERS REQUIRED_VARS XDR_INCLUDE_DIR)
+  if(XDR_HEADERS_FOUND)
+    option(LAMMPS_XDR "include XDR compatibility files for doing particle dumps in XTC format" OFF)
+  else()
+    option(LAMMPS_XDR "include XDR compatibility files for doing particle dumps in XTC format" ON)
+  endif()
+  if(NOT XDR_HEADERS_FOUND AND NOT LAMMPS_XDR)
+    message(FATAL_ERROR "MISC package needs xdr headers! Set LAMMPS_XDR=ON to use internal headers.")
+  elseif(XDR_HEADERS_FOUND)
+    include_directories(${XDR_INCLUDE_DIR})
```

Comment:
what's the dropin replacement for it? libtirpc? If so, shouldn't something be linked in as well?

Reply:
LAMMPS contains compatibility software (imported from gromacs) in `xdr_compat.h` and `xdr_compat.cpp`. Environments w/o XDR support are not new, Windows being one of them.
So this should not only apply to future Linux distributions, but should also be tested on other non-XDR OS environments.